content/v0.80.0.md

```diff
@@ -0,0 +1,478 @@
+- #2090: Refactor the ReelMagic audio FIFO class
+- #2086: Refactor ReelMagic player with per-stream handle registration
+- #2045: Split contributed CD-DA scripts into fetch vs. cue functions
+- #2060: Minor mouse enhancements
+- #2117: Use shared pointers to manage the virtual files
+
+
+</details>
+
+##  6. <a name='Bugandregressionfixes'></a>Bug and regression fixes
+
+- The mixer's effects (reverb, chorus, crossfeed) can now be turned on and off
+  at runtime via the mixer's CLI.
+
+- FluidSynth's specific reverb and chorus configuration settings are now applied
+  and will replacing the default values that were previously always applied.
+
+- The modem's call now has a warm-up period where it will discard data. This
+  fixes a bug noted in Duke Nukem 3D where it would fail it's initial handshake
+  in the first try.
+
+<details>
+  <summary>Log of 0.80's pull requests involving bug and regression fixes:</summary>
+- #2143: Capture the OpenGL screenshot using RGBA pixel format
+- #2139: Always use VFR presentation mode for text modes
+- #2133: Fix FluidSynth reverb and chorus settings not applying
+- #2125: Add modem call warm-up period
+- #2124: Drop architecture spec from setup-msbuild GitHub action
+- #2119: Remove errorneous screen parameters check in mouse code
+- #2115: Avoid shifting negative DAC data in Sound Blaster #2113 (CWE-758)
+- #2110: Fix assertion in mouse code for surface screen output
+- #2102: Fix tab handling in MORE command
+- #2094: Fix character D6 mapping in code page 3021 (Bulgarian MIK encoding)
+- #2091: Restore WheelHasMoved bit in DOS mouse driver to correct value
+- #2087: Restore 0.79.1 mouse acceleration model in DOS driver
+- #2083: MORE text output engine fixes
+- #2038: Initialize the window's canvas size during its setup phase
+- #2034: Fix bitwise shift of negative in DOS date pack call
+- #2021: Improve `mixer.com` CLI handling of volume and effect arguments
+- #2011: Adapt 'update-sources.sh' script to UTF-8 locale
+- #1995: Use non-throwing form of `std::filesystem` calls
+- #1999: Limit `bit_view`s to 8-bit registers to avoid byte-order assumption
+- #1985: Fix a memory leak in the keyboard layout class
+- #2001: Parse language options before writing the conf file
+- #1998: Fix several English strings too long to fit on screen
+- #1991: Change 'config -wc' to write UTF-8 file
+</details>
+
+##  7. <a name='Translationimprovements'></a>Translation improvements
+
+Translations:
+* German: up-to-date
+* Italian: up-to-date
+* Polish: up-to-date
+* Russian: up-to-date
+* French: not updated
+* Spanish: not updated
+
+Related improvements:
+
+- The title bar is now translatable.
+
+- The Brazilian ABNT2 slash and question mark keys are now handled.
+
+- Translations are now stored in UTF-8 and converted to the selected DOS
+  codepage, dropping accents if needed, sometimes even performing crude
+  transliterations - for Latin, Greek, and Cyrillic alphabets emulator shouldn't
```

Comment:
change `emulator` to `DOSBox`

Reply:
Replaced the line with: `  Note that only DOSBox Staging's own messages benefit from this feature.`